gulpfile.js

```diff
@@ -68,8 +68,10 @@ function test() {
     .pipe($.mocha({reporter: 'dot', globals: config.mochaGlobals}));
 }
 
+// Make babel preprocess the scripts the user tries to import from here on.
+require('babel/register');
```

Comment:
Main reason is just to prevent watch from attempting registration multiple
times - several ways to approach that issue, this is the simplest

On Mon, Mar 23, 2015 at 3:07 PM, James Smith <notifications@github.com>
wrote:

> In gulpfile.js
> <https://github.com/jmeas/es6-node-boilerplate/pull/5#discussion_r26968603>
> :
>
> > @@ -68,8 +68,10 @@ function test() {
> >      .pipe($.mocha({reporter: 'dot', globals: config.mochaGlobals}));
> >  }
> >
> > +// Make babel preprocess the scripts the user tries to import from here on.
> > +require('babel/register');
>
> Do we want to move this outside of the tasks? [image: :open_mouth:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmeas/es6-node-boilerplate/pull/5/files#r26968603>.
>